doc/man3/RSA_get0_key.pod

```diff
@@ -36,6 +45,13 @@ private key (see PKCS#1 section 3 Key Types), where B<p> and B<q> are
 the first and second factor of B<n> and B<dmp1>, B<dmq1> and B<iqmp>
 are the exponents and coefficient for CRT calculations.
 
+For multi-prime RSA (defined in RFC 8017), there are also one or more
+'triplet' in an RSA object. A triplet contains three members, B<r>, B<d>
+and B<t>. B<r> is the additional prime besides B<p> and B<q>. B<d> and
+B<t> are the exponent and coefficient for CRT calculations.
+RSA_get_multi_prime_extra_count() returns the number of the triplets
+which is also the number of other primes in the given RSA object B<r>.
```

Comment:
This sentence doesn't really belong. It's should either be removed or reworded as forward reference.

Reply:
removed